Add retry_on_failure/sending_queue config to values.yaml (#460)
signalfx/splunk-otel-collector-chart: CHANGELOG.md

## Unreleased

### Added

> - Add retry_on_failure/sending_queue config to values.yaml (#460)

Reviewer: ```suggestion
- Add `splunkPlatform.retryOnFailure` and `splunkPlatform.sendingQueue` config options to values.yaml (#460)
```
